day3/README.md

```diff
@@ -44,13 +44,97 @@ React.render(<Game data={ data } />, document.body);
 
 In this extract, both Score and Game are view components that we used last time. The Game component are the one being rendered by React, and this component is using the Score component. In the render function, we are passing the data object as a prop to the Game component, which are passed on down to the Score component. At last the Score component are using the receiving prop to show the score in a div.
 
```

Comment:
In this extract, both Score and Game are view components that we used last time. The Game component is the one being rendered by React, and this component uses the Score component. In the render function, we are passing the data object as a prop to the Game component, which is passed on down to the Score component. At last the Score component uses the receiving prop to show the score in a div.